neqo-transport/src/ecn.rs

```diff
@@ -225,20 +227,33 @@ impl Info {
             && (self.baseline - prev_baseline)[IpTosEcn::Ce] > 0
     }
 
+    /// An [`IpTosEcn::Ect0`] marked packet has been acked.
+    pub(crate) fn acked_ecn(&mut self) {
+        if let ValidationState::Testing {
+            initial_probes_acked: probes_acked,
+            ..
+        } = &mut self.state
+        {
+            *probes_acked += 1;
```

Comment:
Should we immediately declare success when this happens?

Reply:
We certainly could.

I read the recommendation in RFC 9000 in a way that suggests to wait for more than one signal before declaring a path as _capable_:

> The goal is not to limit the duration of the testing period but to ensure that enough marked packets are sent for received ECN counts to provide a clear indication of how the path treats marked packets. [Section 13.4.2](https://www.rfc-editor.org/rfc/rfc9000.html#ecn-validation) suggests limiting this to ten packets or three times the PTO.

https://www.rfc-editor.org/rfc/rfc9000.html#ecn-alg

Do you expect paths to bleach intermittently? If so, I believe a testing period of 10 packets, will give us more accurate statistics.

Thoughts @larseggert?

Reply:
Sure. But I really wonder if there is intermittent bleaching anywhere.